spec/consensus/time.md

```diff
@@ -0,0 +1,52 @@
+---
+order: 2
+---
+# Time
+
+CometBFT provides a Byzantine fault-tolerant source of time.
+
+Time in CometBFT is defined with the [`Time`][spec-time] field of the
+block [`Header`][spec-header].
+
+## Properties
+
+The Time produced by CometBFT satisfies the following properties:
+
+- **Time Monotonicity**: time is monotonically increasing.  More precisely, given
+  two block headers `H1` of height `h1` and `H2` of height `h2`,
+  it is guaranteed that if `h2 > h1` then `H2.Time > H1.Time`.
+
+- **Byzantine Fault Tolerance**: faulty or malicious processes should not be able
+  to arbitrarily influence (increase or decrease) the block Time.
+  In other words, the Time of blocks should be defined by correct processes.
```

Comment:
I have a (small) problem with this sentence. If a byzantine process (which is byzantine for the whole run) decides to use a "correct" time value, this sentence is not correct, is it?
I know it can't happen in BFT-Time, but it _could_ in PBTS if the byzantine node in question is the proposer.

Reply:
In this case they are not arbitrarily influencing the block time, are them?

Reply:
Agreed, that's about the first sentence, which I'm fine with.
The problem is the second sentence, which says "the Time of blocks should be defined by correct processes".

Reply:
As is starts with "In other words...", consider removing it